newdle/notifications.py

```diff
@@ -30,15 +30,35 @@ def notify_newdle_participants(
     return send_emails(emails)
 
 
+def notify_newdle_creator(participant, subject, text_template, html_template, context, attachments=None):
+    if not attachments:
+        attachments = []
+    creator_email = participant.newdle.creator_email
+    sender_name = current_app.config['NOREPLY_ADDRESS']
```

Comment:
What about using the name of the participant here? Having an email address in the name part of the email is rather strange IMO.